app/controllers/frontend/events_controller.rb

```diff
@@ -29,6 +29,11 @@ def related_events(n)
     def load_event
       @event = Frontend::Event.find_by!(slug: params[:slug])
       @conference = @event.conference
+      @player = ''
+      if params[:player]
+        # TODO input sanitation/validation?
```

Comment:
yes, at least check against a whitelist and the code should go into the controller.